src/utils/utils.js

```diff
@@ -187,9 +187,9 @@ module.exports = {
   },
 
   /*
-  * Returns the path where the blockchain data directory is, and also creates the directory if it doesn't exist.
+  * Returns the path where the blockchain version directory is.
   */
-  getBlockchainDataDir: () => {
+  getBlockchainVersionDir: () => {
```

Comment:
lets call this `getVersionDir` since its a combination of contract and db schema numbers